package.json

```diff
@@ -9,7 +9,8 @@
     "test": "prettier --check . && jest src --coverage",
     "test:debug": "prettier --check . && node --inspect node_modules/.bin/jest --runInBand --coverage src",
     "preversion": "npm test",
-    "version": "npm run build"
+    "version": "npm run build",
+    "prepare": "rm -rf ./dist/* && prettier --check  . && ./node_modules/typescript/bin/tsc && webpack --config webpack.prod.js"
```

Comment:
nit: just use the tsc command directly instead of referencing the node modules

Also, you can use prehooks in npm to run things before other things, unless you want to run prepare on its own. I.e. you can make it run prepare every time before an npm start https://www.yld.io/blog/using-npm-pre-and-post-hooks/

Reply:
Since this is the exact same thing as build I'll change it to call that.

Based on this it seems like prepare is something we'd want to implement: https://docs.npmjs.com/cli/v7/using-npm/scripts#life-cycle-scripts